.gitlab-ci.yml

```diff
@@ -4,7 +4,7 @@ stages:
 # Global default parameters set for all jobs unless overridden by job-specific
 # configuration.
 default:
-  image: python:3.6-buster
+  image: python:3.8-buster
```

Comment:
I don't think the use of the `...-buster` naming here is necessary. I'm not sure why I did it in the first place and the image name `python:3.8` is wholly valid.